charts/airbyte/Chart.yaml

```diff
@@ -29,14 +29,6 @@ dependencies:
     tags:
       - bitnami-common
     version: 1.x.x
-  - condition: postgresql.enabled
-    name: postgresql
-    version: 10.x.x
-    repository: https://charts.bitnami.com/bitnami
-  - condition: global.logs.minio.enabled
-    name: minio
-    version: 11.x.x
-    repository: https://charts.bitnami.com/bitnami
```

Comment:
is this related to fixing tests or is this just a separate change we want to include in this PR as well?